main.go

```diff
@@ -589,6 +589,32 @@ func main() {
 				}
 				return cfg.Save(configFile)
 			},
+		}, {
+			Name:  "shuttle",
+			Usage: "Initializes a shuttle node, returns handle and authorization token",
+			Action: func(cctx *cli.Context) error {
+				configFile := cctx.String("config")
+				if err := cfg.Load(configFile); err != nil && err != config.ErrNotInitialized { // still want to report parsing errors
+					return err
+				}
+				shuttle := &model.Shuttle{
```

Comment:
move this model to the where it gets created in the db

Reply:
think i understood